modules/logging-loki-zone-fail-recovery.adoc

```diff
@@ -65,21 +65,21 @@ wal-logging-loki-ruler-1
 oc delete pvc __<pvc_name>__  -n openshift-logging
 ----
 +
-. Then delete the pod(s) by running the following command:
+. Delete the pod(s) by running the following command:
 +
 [source,terminal]
 ----
 oc delete pod __<pod_name>__  -n openshift-logging
```

Comment:
There are some spots in the preview assembly where commands are either missing their `$` prompt, are using the non-openshift-docs style for user-replaceables (e.g., `__<pvc_name>__`), or both. Please check throughout the new built assembly and adjust the related modules that need fixes (some appear to not be included in this PR yet).

E.g., these ones should look like:
```
$ oc delete pvc <pvc_name> -n openshift-logging
```
and
```
$ oc delete pod <pod_name> -n openshift-logging
```